DQM/L1TMonitorClient/python/L1TStage2BMTFQualityTests_cfi.py

```diff
@@ -0,0 +1,15 @@
+# quality tests for L1T Stage2 BMTF 
+ 
+import FWCore.ParameterSet.Config as cms
+
+l1TStage2BMTFQualityTests = cms.EDAnalyzer("QualityTester",
```

Comment:
@davidlange6 Not yet. Eventually, QualityTester should be migrated into a harvester, eventually...
But this is out of scope for this PR.